res/database/Default/user/scr0llbaer.xml

```diff
@@ -0,0 +1,1586 @@
+				<job index="8" function="32" required="1" gender="1" />
+				<job index="9" function="32" required="1" gender="2" />
+			</jobs>
+
+			<data flags_optional="40" />
+			<genres maingenre="16" subgenres="5,14" />
+			<blocks value="3" />
+			<price min="31000" max="48000" />
+			<potential min="55" max="92" />
+			<outcome min="67" max="89" />
+			<review min="77" max="92" />
+			<speed min="41" max="73" />
+			<studio_size value="3" />
+		</scripttemplate>
+
+		<scripttemplate product="1" licence_type="1" guid="40d169cd-eeb2-400e-b839-f69690893bc1" creator="9551" comment="Quelle: Das Kino in meinem Kopf, das enstand, als ich zum ersten Mal den Titel von TheRob-Script-Com-EngeloderBengel, bei mir 'Aribert außer Rand und Band' gelesen hatte. :)">
+			<title>
+				<de>%title%</de>
+				<en>%title%</en>
+			</title>
+			<description>
+				<de>%name% ist behütet in einer fürsorglichen Familie aufgewachsen und lebt Mitte 20 in einer Studenten-WG mit braven Kommilitonen. Doch er spürt, etwas ist in ihm. Was ist das nur? Ist er vielleicht nicht ganz monogam? Sein Umfeld beginnt zu verzweifeln...</de>
+				<en>%name% grew up safely in a caring family and in his mid-20s lives in a flat-share with well-behaved fellow students. But he senses something is inside him. What is it? Is he perhaps not quite monogamous? Everyone close to him begins to despair...</en>
+			</description>
+			<variables>
+				<title>
+					<de>%name% vogelwild|%name% ganz aus dem Häuschen|%name% voll aufgekratzt|Für %name% gibt es kein Morgen mehr|%name% nicht mehr zu bändigen|%name% ganz ganz wild|%name% völlig ungezügelt|%name% wird ganz wild</de>
+					<en>%name% all boisterous|%name% all over the place|%name% totally psyched|For %name% there is no more tomorrow|%name% no longer to be tamed|%name% going so very wild|%name% totally unbridled|%name% gets all wild</en>
+				</title>
+				<name>
+					<de>%PERSONGENERATOR_FIRSTNAME(de,1)%|%PERSONGENERATOR_FIRSTNAME(es,1)%</de>
+				</name>
+			</variables>
+			
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="2" required="1" gender="1" />
+				<job index="2" function="2" required="1" gender="2" />
+				<job index="3" function="2" required="1" gender="1" />
+				<job index="4" function="2" required="1" gender="2" />
+				<job index="5" function="2" required="1" gender="1" />
+			</jobs>
+			<genres maingenre="5" subgenres="9,7" />
+			<blocks min="2" max="3" />
+			<price min="60000" max="111500" />
+			<potential min="65" max="100" />
+			<outcome min="65" max="100" />
+			<review min="65" max="100" />
+			<speed min="65" max="100" />
+			<studio_size min="2" max="3" />
+		</scripttemplate>
+
+		<scripttemplate product="2" licence_type="3" guid="ea871c60-e653-4dac-82bb-9e2e78393385" creator="9551" comment="Was available as a TV series in MadTV - https://en.wikipedia.org/wiki/Xanth">
+			<title>
+				<de>Die %saga% vom %magic% Xynthia (Staffel 1)</de>
```

Comment:
No because i was intending to add more seasons eventually, and then that would be confusing.

Makes me think we may also need a new type in the database: places, so that these can be referenced. Or maybe a a general object type should suffice.